.github/runbooks.yml

```diff
@@ -50,3 +50,4 @@ runbooks:
         upp-concept-carousel-queue: runbooks/upp-concept-carousel-queue.md
         upp-ontotext-backup-aurora: runbooks/upp-ontotext-backup-aurora.md
         upp-concept-events-queue: runbooks/upp-concept-events-queue-runbook.md
+        concept-elasticsearch-cluster: concept-elasticsearch-cluster-runbook.md
```

Comment:
Hm, I was surprised that the bot pick it up like that i.e. missing `runbooks/` prefix. Maybe add it just to be consistent with the other entries.

Reply:
Good catch. I'll fix this.